eogrow/core/area/base.py

```diff
@@ -301,4 +301,4 @@ def _add_bbox_column(grid: List[gpd.GeoDataFrame]) -> None:
         """Adds a column with bounding boxes to all dataframes in a grid"""
         for bbox_df in grid:
             crs = CRS(bbox_df.crs)
-            bbox_df["BBOX"] = bbox_df.geometry.apply(lambda geo: BBox(geo.bounds, crs))
+            bbox_df["BBOX"] = bbox_df.geometry.apply(lambda geo: BBox(geo.bounds, crs))  # noqa B023
```

Comment:
> B023: Functions defined inside a loop must not use variables redefined in the loop, because [late-binding closures are a classic gotcha](https://docs.python-guide.org/writing/gotchas/#late-binding-closures).

is this just because there is a lambda function here?

Reply:
yes. It is not dangerous in our case because it is applied instantly, but python closures are horrifying.